docs/tutorial_basics.rst

```diff
@@ -789,6 +779,28 @@ If any obtained performance value is beyond its respective thresholds, the test
      * Rerun with '-n StreamWithRefTest -p gnu --system catalina:default'
      * Reason: performance error: failed to meet reference: Copy=24586.5, expected 55200 (l=52440.0, u=57960.0)
 
+Also, note how the performance syntax for this example is far more compact in comparison to our first iteration of the STREAM test.
```

Comment:
I would provide that in a separate version of the stream test (so as to be able to test it as well). At the beginning of the paragraph we are talking only about the reference and here we show a completely different implementation of the performance variables extraction. I would add all this to a subsection named perhaps "setting explicitly the test's performance variables."

Reply:
Makes sense 👍